### PR TITLE
docs(console): Add cluster autopause

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Update the following pages to include content for the new cluster autopause feature.
* [Clusters](https://docs.camunda.io/docs/next/components/concepts/clusters/)
* [Resume your cluster](https://docs.camunda.io/docs/next/components/console/manage-clusters/resume-cluster/) 

Content updated for following doc versions: 
* Next
* 8.5
* 8.4
* 8.3
* 8.2

Relates to [Autopause dev clusters after 1 day #2419](https://github.com/camunda/product-hub/issues/2419).

## When should this change go live?

Note: Potentially to go live as part of the 8.6-alpha-4 release, tbc by Engineering.

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/concepts/clusters.md [hunk 1]
```diff
@@ -53,6 +53,36 @@ Additionally in the Starter Plan, the following applies to **development cluster
...
+| Free         | 8 hours             | _exempt_ | _exempt_ | _exempt_ |
+| Starter      | _exempt_            | _exempt_ | _exempt_ | _exempt_ |
+| Professional | _exempt_            | _exempt_ | _exempt_ | _exempt_ |
+| Enterprise   | _exempt_            | _exempt_ | _exempt_ | _exempt_ |
+
+\* Untagged clusters are treated as `dev` tagged clusters.
+
+:::tip
+
+To avoid having to keep resuming your cluster(s) after an autopause, you can:
```
Comment (ultraschuppi) [inline docs/components/concepts/clusters.md:82]: i'd say it is more about avoiding that my cluster falls asleep ;-)

**Review by ultraschuppi — APPROVED** [review]
lgtm

File: docs/components/concepts/clusters.md (hunk 1 above)
Comment (mesellings) [reply docs/components/concepts/clusters.md]: Thanks @ultraschuppi, agreed about reframing it - I have changed that line to "To avoid your clusters being automatically paused, you can:"

File: docs/components/concepts/clusters.md [hunk 2]
```diff
@@ -53,6 +53,36 @@ Additionally in the Starter Plan, the following applies to **development cluster
...
+
+Free and Free Trial plan `dev` (or untagged) clusters are automatically paused 8 hours after the creation of the cluster or since you last resumed it from a paused state.
+
+No data is lost while a cluster is paused. All execution and configuration is saved, but cluster components such as Zeebe and Operate are temporarily disabled until you resume the cluster.
+
+- Production clusters and clusters tagged as `test`, `stage`, or `prod` are exempt and do not autopause.
+- Starter, Professional, and Enterprise customer clusters are exempt and do not autopause.
+- Paused clusters are automatically deleted after 30 consecutive days in a paused state. You can change the tag to avoid cluster deletion.
+- Autopause occurs regardless of cluster usage.
+- You can resume a paused cluster at any time. See [Resume your cluster](/components/console/manage-clusters/resume-cluster.md/).
```
Comment (conceptualshark) [inline docs/components/concepts/clusters.md:66]: Re: the above comment on autopause, I feel like a lot of important information is competing for real estate on this bulleted list with less demanding info. Maybe "You can resume a paused cluster" could be moved to the summary - and the information on starter/professional/enterprise might be less necessary if the content is moved beneath the Free Trial h2?

**Review by conceptualshark — COMMENTED** [review]
Non-blocking Q's/suggestions here! 

Considering the current structure of this page, is it viable to pare some of this down and move it into an h3 under the Free Trial section? If this is only really relevant for free account users, it could clean up some of the extra info.

File: docs/components/concepts/clusters.md [hunk 3]
```diff
@@ -53,6 +53,36 @@ Additionally in the Starter Plan, the following applies to **development cluster
 - **Cluster is not highly available & includes less hardware**: Reduced hardware resources and availability compared to production cluster (for example, one Zeebe node only).
 - **Shorter history of processes and decisions**: Data retention in Operate, Optimize, and Tasklist is reduced to one day. For example, pending or historical process instances are deleted after one day as per the [fair usage limits of the Starter plan](https://camunda.com/legal/fair-usage-limits-for-starter-plan/).
 
-:::caution
-**Cluster auto-pause** is not yet available and only applies to non-Enterprise clusters. Development clusters will be paused if they go unused for two hours. When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events.
+## Autopause
+
+Free and Free Trial plan `dev` (or untagged) clusters are automatically paused 8 hours after the creation of the cluster or since you last resumed it from a paused state.
```
[the remark this answers is not in the document]
Comment (mesellings) [reply docs/components/concepts/clusters.md]: @conceptualshark Great feedback, agree completely - I wrestled over whether to include this in the Free Trial section, but originally felt it was too much for that, but with some editing it does fit much better there - thanks! 🥇

File: docs/components/concepts/clusters.md (hunk 2 above)
Comment (mesellings) [reply docs/components/concepts/clusters.md]: Agreed, great feedback, I've incorporated your suggestions into a new version. 👍

**Comment by mesellings** [comment]
Incorporated feedback in Slack channel discussion: https://camunda.slack.com/archives/C07E7N3G865/p1722506582553509?thread_ts=1722502951.388239&cid=C07E7N3G865

**Comment by akeller** [comment]
@mesellings based on your comment 
> 8.6-alpha-4 release

I added the alpha4 label. We can always remove it, but it's a good reminder to prompt if it's ready to go with the alpha.

**Comment by mesellings** [comment]
> @mesellings based on your comment
> 
> > 8.6-alpha-4 release
> 
> I added the alpha4 label. We can always remove it, but it's a good reminder to prompt if it's ready to go with the alpha.

Thank you, I was undecided about whether to add it (I could add a potential label in the issue) but I will try and get a better idea next week how quickly engineering are doing it.

**Review by conceptualshark — DISMISSED** [review]
👌 This looks really clean to me!

**Review by christinaausley — DISMISSED** [review]
This looks great! My only nitpick is spelling out numbers under 10, and leaving 10+ as numerics 😄 Could also always just say "which typically takes less than 10 minutes" or "5-10 minutes", and "automatically paused eight hours" -- non-blocking, but a little tiny detail.

**Comment by mesellings** [comment]
> This looks great! My only nitpick is spelling out numbers under 10, and leaving 10+ as numerics 😄 Could also always just say "which typically takes less than 10 minutes" or "5-10 minutes", and "automatically paused eight hours" -- non-blocking, but a little tiny detail.

Thank you - I went back and forth on these numbers, and decided that I couldn't write "five to 10" (mix) so felt "five to ten" was better/consistent, but I did miss the eight/8 hours - well spotted! I will change that to "eight", but leave "five to ten" as the five is important (it is normally five, but we wanted to be cautious and say up to ten).

As an interesting aside, in UX writing and generally now in web tech writing, there is a move to writing out numbers numerically as it is easier to scan on a web page (e.g. https://www.nngroup.com/articles/web-writing-show-numbers-as-numerals/) and I feel this could be true here, as "8" in the page (which is a vital fact) is easier to pick up than having to 'read' for the number.

TL;DR: I've changed to eight for style consistency as you say (good spot!) but wonder if we might want to revisit numbers in the style guide at some point in light of modern practice? 👍

**Comment by mesellings** [comment]
Think it's good to go now, please approve when you can @christinaausley, @HanselIdes, @camundom.

@ultraschuppi / @HanselIdes  is this looking good to go out next week in the alpha release?

**Comment by mesellings** [comment]
@HanselIdes I have also backported the auto-pause docs to 8.2, 8.3, 8.4, and 8.5 as well as Next, as per this conversation: https://camunda.slack.com/archives/C07E7N3G865/p1723026184304529.

File: docs/components/console/manage-clusters/resume-cluster.md [hunk 4]
```diff
@@ -1,7 +1,17 @@
...
 title: Resume your cluster
-description: "Follow these instructions to resume your cluster"
+description: "You can resume your paused cluster at any time"
 ---
 
-A trial cluster can be paused automatically due to inactivity. To resume your cluster, navigate to the **Clusters** tab, click on the cluster name (e.g. Prod Cluster - Project A) from the list and in the cluster details, and click **Resume cluster**.
+If your cluster is paused, you can resume it at any time.
+
+To resume your paused cluster:
+
```
Comment (HanselIdes) [inline docs/components/console/manage-clusters/resume-cluster.md:10]: This is the long way to achieve this goal.

It's a better UX to use the Web Modeler's deploy modal. There's a button that says "resume" cluster.

image

**Review by HanselIdes — DISMISSED** [review]
Hi @mesellings , this looks good to me and can ship without further changes

(Optional) I added an alternative way to resume the cluster that's much easier and fits better into the user flow.

This change should go out in sync with the engineering effort (https://github.com/camunda/product-hub/issues/2419)

File: docs/components/concepts/clusters.md [hunk 5]
```diff
@@ -24,12 +24,31 @@ When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DM
...
-Free Trial clusters have the same functionality as a production cluster, but are size small and only available during your trial period. You cannot convert a Free Trial cluster to a different kind of cluster.
+Free Trial clusters have the same functionality as a production cluster, but are size [small (S)](/components/best-practices/architecture/sizing-your-environment.md#camunda-8-saas) and only available during your trial period. You cannot convert a Free Trial cluster to a different kind of cluster.
 
 Once you sign up for a Free Trial, you are able to create one production cluster for the duration of your trial.
 
 When your Free Trial plan expires, you are automatically transferred to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster created during your trial is deleted, and you cannot create new clusters.
 
+### Auto-pause
+
+Free Trial `dev` (or untagged) clusters are automatically paused eight hours after the creation of the cluster or since you last resumed the paused cluster. Auto-pause occurs regardless of cluster usage.
```
[the remark this answers is not in the document]
Comment (mesellings) [reply docs/components/concepts/clusters.md]: Change made 👍

File: docs/components/console/manage-clusters/resume-cluster.md (hunk 4 above)
Comment (mesellings) [reply docs/components/console/manage-clusters/resume-cluster.md:26]: I have added in this detail to the Resume cluster page - thanks for highlighting 👍

**Comment by mesellings** [comment]
@HanselIdes I have updated the Resume cluster topic to include your suggestion, thank you 👍

**Review by HanselIdes — APPROVED** [review]
LGTM

**Review by conceptualshark — APPROVED** [review]

**Comment by mesellings** [comment]
Changed target alpha to alpha 5 September release following this Slack conversation: https://camunda.slack.com/archives/C07E7N3G865/p1723191147820799
